docs/how-to-guides/others/planning-evaluation-using-scenarios.md

```diff
@@ -0,0 +1,297 @@
+  features. Therefore, if there are any items that cannot be supported or are difficult to edit using the GUI, you can
+  edit and adjust the scenario directly using a text editor. In that case, the schema of the scenario format is checked
+  before saving, so you can edit with confidence.
+- Best way to understand other complex scenario implementations is to look at the existing scenarios. You can check the
+  scenarios in the
+  [Autoware Evaluator](https://evaluation.ci.tier4.jp/) to see other implementations.
+
+## Running Scenarios using Scenario Simulator
+
+### Creating the Scenario Simulator Environment
+
+- Clone the Autoware main repository.
+
+```bash
+git clone https://github.com/autowarefoundation/autoware.git
+```
+
+- Create source directory.
+
+```bash
+cd autoware
+mkdir src
+```
+
+- Import the necessary repositories both for the Autoware and Scenario Simulator.
+
+```bash
+vcs import src < autoware.repos
+vcs import src < simulator.repos
+```
+
+- If you are installing Autoware for the first time, you can automatically install the dependencies by using the
+  provided Ansible script.
+
+```bash
+./setup-dev-env.sh
+```
+
+- Install dependent ROS packages.
+
+Autoware and Scenario Simulator require some ROS 2 packages in addition to the core components. The tool rosdep allows
+an automatic search and
+installation of such dependencies. You might need to run rosdep update before rosdep install.
+
+```bash
+source /opt/ros/humble/setup.bash
+rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+```
+
+- Build the workspace.
+
+```bash
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+
+- If you have any issues with the installation, or if you need more detailed instructions, please refer to the
+  [Autoware source installation page](https://autowarefoundation.github.io/autoware-documentation/main/installation/autoware/source-installation/).
+
+### Execute the scenario by using Scenario Simulator
+
+We created an example scenario above and exported it into our local machine. Now, we will execute this scenario using
+Scenario Simulator.
+
+- Firstly, we should define the `lanelet2_map.osm` and `pointcloud_map.pcd` files in the scenario file. In default, the
+  configuration is below:
+
+```yaml
+RoadNetwork:
+  LogicFile:
+    filepath: lanelet2_map.osm
+  SceneGraphFile:
+    filepath: lanelet2_map.pcd
+```
+
+User should change the `filepath` field according to the map file we downloaded before. You can find
+both `lanelet2_map.osm` and `pointcloud_map.pcd` files in the `Maps` section of the Autoware Evaluator.
+
+**Note:** Although the `pointcloud_map.pcd` file is mandatory, it is not used in the simulation. Therefore, you can
+define a dummy file for this field.
+
+- After defining the map files, we can execute the scenario by using the Scenario Simulator. The command is below:
+
+```bash
```

Comment:
We can add here as well a link to the tutorial page for scenario simulation as a reference.

Reply:
I didn't want to refer there because its bash command includes `architecture_type:=awf/universe \` but currently it does not work.